scipy/signal/signaltools.py

```diff
@@ -255,6 +254,101 @@ def correlate(in1, in2, mode='full', method='auto'):
                          " 'direct', or 'fft'.")
 
 
+def correlation_lags(in1_len, in2_len, mode='full'):
+    r"""
+    Calculates the lag / displacement indices array for 1D cross-correlation.
+
+    Parameters
+    ----------
+    in1_size : array size
+        First input size.
+    in2_size : array size
+        Second input size.
+    mode : str {'full', 'valid', 'same'}, optional
+        A string indicating the size of the output:
+
+        See the documentation `correlate` for more information.
+
+    See Also
+    --------
+    correlate : Used in conjunction with correlation_lags to produce a
+                1D cross-correlation.
```

Comment:
I don't understand the conjunction point. `correlate` on its own calculates the cross-correlation.

```suggestion
    correlate : Compute the N-dimensional cross-correlation.
```